src/Controls/src/Core/HandlerImpl/Window/Window.Impl.cs

```diff
@@ -34,6 +34,7 @@ public Window()
 			ModalNavigationManager = new ModalNavigationManager(this);
 			Navigation = new NavigationImpl(this);
 			InternalChildren.CollectionChanged += OnCollectionChanged;
+			HighlightLayer = new HighlightLayer(this);
```

Comment:
Is it worth adding to the services instead of the ctor? This way you would do `window.Handler.Services.GetService<IHL>()` and this guarantees you have a native view.